roles/sap_anydb_install_oracle/README.md

```diff
@@ -1,54 +1,71 @@
+<!-- BEGIN Title -->
```

Comment:
With all these section markers this text looks like automatically generated, but I do not see anywhere in PR code to automatically generate roles documentation. I think managing this manually is prone to error.

Reply:
See comment above, this is in preparation for future git action to load it from arugment specs file

Reply:
> With all these section markers this text looks like automatically generated, but I do not see anywhere in PR code to automatically generate roles documentation. I think managing this manually is prone to error.

We are including these section markers for easier modification of the README.md files when creating the downstream versions. It also gives us the opportunity of inserting content from other files where possible.

Also the section markers allow us for detecting if a section is required (in this case, the section header is outside of the section text) or optional (comments are surrounding the section header and the section text).

Besides that, I see no disadvantage in using a template with section markers even when not using any automation, as long as we have a template which we will use and demand to be mandatory.